openjdk/excludes/ProblemList_openjdk21.txt

```diff
@@ -382,6 +385,10 @@ compiler/codegen/aes/TestAESMain.java https://github.com/adoptium/aqa-tests/issu
 
 # jdk_jfr
 
+jdk/jfr/event/runtime/TestDumpReason.java https://github.com/adoptium/aqa-tests/issues/4155
+jdk/jfr/event/runtime/TestShutdownEvent.java https://github.com/adoptium/aqa-tests/issues/4155
+jdk/jfr/jvm/TestDumpOnCrash.java https://github.com/adoptium/aqa-tests/issues/4155
```

Comment:
There is no platform specified

Reply:
updated PTAL, I wasn't sure if I should remove these excludes from the Microsoft vendor problemList or not?

Reply:
These excludes should be removed from the Microsoft vendor problemList. There is no need to duplicate them. openjdk/excludes/ProblemList_openjdkxxx.txt applies to JDK_IMPL=hotspot (regardless of vendor).

Reply:
I've removed the duplicated excludes from the Msft vendor folder